supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesParameterConversions.scala

```diff
@@ -72,42 +67,48 @@ trait PipelinesParameterConversions {
-          ActionBuilder
+          val localizationAction = ActionBuilder
             .withImage(image)
             .withCommand("/bin/sh", "-c", s"$createNgc; mkdir $mountpoint; $runFusera")
             .withMounts(mounts)
             .withFlags(List(ActionFlag.RunInBackground, ActionFlag.EnableFuse))
+          List(ActionBuilder.describeParameter(fileInput, labels), localizationAction)
         case _: HttpPath =>
           val dockerImage = "google/cloud-sdk:slim"
           val command = s"curl --silent --create-dirs --output ${fileInput.containerPath} ${fileInput.cloudPath}"
-          ActionBuilder
+          val localizationAction = ActionBuilder
             .withImage(dockerImage)
             .withCommand("/bin/sh", "-c", command)
             .withMounts(mounts)
             .withLabels(labels)
             .setEntrypoint("")
-        case _ => cloudSdkShellAction(localizeFile(fileInput.cloudPath, fileInput.containerPath))(mounts, labels = labels)
+          List(ActionBuilder.describeParameter(fileInput, labels), localizationAction)
+        case _: GcsPath =>
+          // GCS paths will be localized with a separate localization script.
+          Nil
       }
-
-      NonEmptyList.of(describeAction, localizationAction)
     }
   }
 
   implicit val directoryInputToParameter = new ToParameter[PipelinesApiDirectoryInput] {
     override def toActions(directoryInput: PipelinesApiDirectoryInput, mounts: List[Mount])
-                          (implicit retryPolicy: LocalizationConfiguration): NonEmptyList[Action] = {
-      val labels = ActionBuilder.parameterLabels(directoryInput)
-      val describeAction = ActionBuilder.describeParameter(directoryInput, labels)
-      val localizationAction = cloudSdkShellAction(
-        localizeDirectory(directoryInput.cloudPath, directoryInput.containerPath)
-      )(mounts = mounts, labels = labels)
-      NonEmptyList.of(describeAction, localizationAction)
+                          (implicit retryPolicy: LocalizationConfiguration): List[Action] = {
+      directoryInput.cloudPath match {
+        case _: GcsPath => Nil // GCS paths will be localized with a separate localization script.
+        case _ =>
+          val labels = ActionBuilder.parameterLabels(directoryInput)
+          val describeAction = ActionBuilder.describeParameter(directoryInput, labels)
+          val localizationAction = cloudSdkShellAction(
+            localizeDirectory(directoryInput.cloudPath, directoryInput.containerPath)
+          )(mounts = mounts, labels = labels)
+          List(describeAction, localizationAction)
+      }
     }
   }
 
   implicit val fileOutputToParameter = new ToParameter[PipelinesApiFileOutput] {
     override def toActions(fileOutput: PipelinesApiFileOutput, mounts: List[Mount])
-                          (implicit retryPolicy: LocalizationConfiguration): NonEmptyList[Action] = {
+                          (implicit retryPolicy: LocalizationConfiguration): List[Action] = {
```

Comment:
This one is still always a `NonEmptyList`

Reply:
not in the next version 😉